app/src/main/java/com/example/fundatecheroes/character/view/NewCharacterActivity.kt

```diff
@@ -8,13 +9,16 @@ import android.widget.AdapterView.OnItemSelectedListener
 import android.widget.ArrayAdapter
 import android.widget.Toast
 import androidx.activity.viewModels
+import androidx.core.view.isVisible
+import com.example.fundatecheroes.R
 import com.example.fundatecheroes.character.presentation.NewCharacterViewModel
 import com.example.fundatecheroes.character.presentation.ViewState
 import com.example.fundatecheroes.databinding.ActivityNewCharacterBinding
+import com.example.fundatecheroes.home.view.HomeActivity
 
 class NewCharacterActivity : AppCompatActivity(), OnItemSelectedListener {
 
-    private var courses = arrayOf<String?>("Herói", "Vilão")
+    private var courses = arrayOf<String?>("Herói ou Vilão? v", "HERO", "VILLAIN")
```

Comment:
Essas strings poderiam estar no arquivo de strings do android
Acredito que não seja preciso esse `v`
```suggestion
    private var courses = arrayOf<String?>("Herói ou Vilão?", "HERO", "VILLAIN")
```